src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -231,7 +234,7 @@ public static List<String> getElementsDisputed(SscsCaseData caseData) {
 
     public static List<String> getNonStandardHearingDurationReasons() {
         // TODO Future Work
-        return Collections.emptyList();
+        return null;
```

Comment:
Is this right ?

Reply:
This was written earlier by Lucas for future work, and I took same changes in, makes sense to return empty list, will revert this.